releasenotes/notes/helm-check-stop-informers-1282dc2111dd8f08.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Fixed a resource leak in the helm check.
```

Comment:
Let's add a copy in releasenotes-dca/notes as well

Reply:
Added :+1: